### DISCUSSION TITLE
SICP in Emacs - Konstantinos Chousos

### DISCUSSION THREAD

**Discussion by giscus** [kchousos/kchousos.github.io#3] [Announcements]
# SICP in Emacs - Konstantinos Chousos

I recently began reading the notorious “Structure and Interpretation of Computer Programs”, a.k.a. the Wizard book. I’m only on the first chapter, but I can already see its value and why it gets recommended so much.

https://kchousos.github.io/posts/sicp-in-emacs/

**Comment by kchousos** [comment]
Reposting as comment instead of a Github issue. By @lockywolf:

>You might find this useful:
>
>https://gitlab.com/Lockywolf/chibi-sicp

**Comment by Eanilsen** [comment]
You could also set the variable `bookmark-save-flag` to either `1` or `0` to automatically save the bookmark each time you make a change.

Thank you for a great article.

**Reply by kchousos** [reply]
Oh, that looks great! Didn't know that, thanks.

**Comment by marcmatsen** [comment]
Hi Konstantinos, thanks for posting this walkthrough. I'm about to take on SICP myself, and it was nice to not immediately get lost in the environment setup weeds. Hope your own process is going well!

**Reply by kchousos** [reply]
You're welcome! Tbh, lately I've been studying for exams and tackling other projects, so SICP was put on hold for a while. But with the new semester starting I'm planning to continue this endeavour once again!

**Comment by goofansu** [comment]
Thank you for the post, I configured the environment successfully. Love the literate programming way.

**Comment by cybertheye** [comment]
how to evoke Racket REPL in emacs?
